src/css/override_components.scss

```diff
@@ -183,3 +159,16 @@ input:focus {
     width: $grid-4;
   }
 }
+
+.fa {
```

Comment:
@thisisdano We're currently only using fontawesome on docs, are you suggesting we use it on dashboard too? I think that requires a larger discussion.